Sources/Sentry/SentryProfiler.mm

```diff
@@ -254,7 +254,7 @@ - (SentryEnvelopeItem *)buildEnvelopeItemForTransaction:(SentryTransaction *)tra
     profile[@"device_physical_memory_bytes"] =
         [@(NSProcessInfo.processInfo.physicalMemory) stringValue];
     profile[@"environment"] = transaction.environment;
-    profile[@"platform"] = transaction.platform;
+    profile[@"platform"] = transaction.platform ?: @"production";
```

Comment:
This doesn't look right, do you mean to put this default value on line 256?

Reply:
It looks like `platform` defaults to `"cocoa"` if we also want to hardcode a fallback here.

Reply:
Hey @indragiek. Are you addressing this?

Reply:
whoops, good catch. sorry